Bynder/Api/Converters/LowercaseConverter.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Bynder.Models;
+
+namespace Bynder.Api.Converters
+{
+    public class LowerCaseConverter : ITypeToStringConverter
+    {
+        public bool CanConvert(Type typeToConvert)
+        {
+            return typeof(AssetType?).IsAssignableFrom(typeToConvert);
```

Comment:
Yes, I agree we can make it more generic. But the .toString method does not have sense for all objects, in most cases it will just return the type of the object. However we can extend it for enums and have something like LowerCaseEnumConverter.